newIDE/app/src/Utils/GDevelopServices/Play.js

```diff
@@ -294,3 +294,83 @@ export const deleteLeaderboardEntry = async (
   );
   return response.data;
 };
+
+// 2 types of comments. Feedback is private, Review is public.
+export type CommentType = 'FEEDBACK' | 'REVIEW';
+
+export type GameRatings = {
+  version: number,
+  visuals: number,
+  sound: number,
+  fun: number,
+  easeOfUse: number,
+};
+
+export type Comment = {
+  id: string,
+  type: CommentType,
+  gameId: string,
+  buildId?: string,
+  text: string,
+  ratings?: GameRatings,
+  playerId?: string,
+  playerName?: string,
+  contact?: string,
+  createdAt: number,
```

Comment:
I think there is also an updatedAt prop. not used, but could be useful in the future